services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/export_csv.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import request from 'request';
+
 import auth_token from '../modules/get_auth_token';
```

Comment:
I think `import auth_token` can be removed along with the `authenticity_token` key further down